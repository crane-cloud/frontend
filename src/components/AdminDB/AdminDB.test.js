import React from "react";
import { render } from "@testing-library/react";

import { Provider } from "react-redux";
import configureStore from "redux-mock-store";
import AdminDBList from "./";
import { useSelector as useSelectorMock } from "react-redux";

// Mock the useSelector hook
jest.mock("react-redux", () => ({
  ...jest.requireActual("react-redux"),
  useSelector: jest.fn(),
}));
describe("AdminDBList", () => {
  const mockStore = configureStore();
  const initialState = {}; // Define your initial state here if needed
  let store;

  beforeEach(() => {
    store = mockStore(initialState);
    useSelectorMock.mockReset();
  });

  test("renders without crashing", () => {
    const wrapper = render(
      <Provider store={store}>
        <AdminDBList />
      </Provider>
    );
  });
});
