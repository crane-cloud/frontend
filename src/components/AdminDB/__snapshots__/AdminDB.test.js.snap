// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminDBList renders without crashing 1`] = `ShallowWrapper {}`;
